sklearn/covariance/graph_lasso_.py

```diff
@@ -557,7 +557,7 @@ def fit(self, X, y=None):
         X : ndarray, shape (n_samples, n_features)
             Data from which to compute the covariance estimate
         """
-        X = check_array(X)
+        X = check_array(X, ensure_min_features=2)
```

Comment:
Again why is this non-default `ensure_min_x` needed here  and in `GraphLasso.fit`? It does not seem related to this PR. I think an inline comment is need to motivate this.